omod/src/main/java/org/openmrs/module/idgen/rest/resource/IdentifierSourceResource.java

```diff
@@ -493,6 +503,39 @@ protected PageableResult doSearch(RequestContext context) throws ResponseExcepti
         return new EmptySearchResult();
     }
     
+    @Override
+    public boolean hasTypesDefined() {
+        return true;
```

Comment:
Overriding this has influenced lots of changes in this PR. Especially those specific to Jackson eg. @JsonIgnore annotations